plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/wc-admin-helper.php

```diff
@@ -262,39 +270,39 @@ public function test_is_store_page() {
 
 		remove_filter( 'pre_option_woocommerce_permalinks', $callback );
 	}
-
-	/**
-	 * Test is_store_page with the defined post_type param.
-	 */
-	public function test_is_store_page_with_post_type() {
-		// Test with post_type=product.
-		$this->assertTrue( WCAdminHelper::is_store_page( 'https://example.com/?post_type=product' ) );
-		// Test with post_type=product and other params.
-		$this->assertTrue( WCAdminHelper::is_store_page( 'https://example.com/test?param1=value1&post_type=product&param2=value2' ) );
-
-		// should return false if post_type is not product.
-		$this->assertFalse( WCAdminHelper::is_store_page( 'https://example.com/test?param1=value1&param2=value2' ) );
-	}
-
-	/** Test product archive link is store page even if shop page not set. */
-	public function test_is_store_page_even_if_shop_page_not_set() {
-		$shop_page_id = get_option( 'woocommerce_shop_page_id' );
-
-		// Unset shop page.
-		add_filter(
-			'woocommerce_get_shop_page_id',
-			function () {
-				return false;
-			},
-			10,
-			1
-		);
-		global $wp_post_types;
-		$wp_post_types['product']->has_archive = 'shop';
-
-		$product_post_type = get_post_type_object( 'product' );
-
-		$link = get_post_type_archive_link( 'product' );
-		$this->assertTrue( WCAdminHelper::is_store_page( $link ) );
-	}
+	//
+	// **
```

Comment:
Hey @moon0326, is this comment out intended?

Reply:
@ilyasfoo My bad 🤦 I forgot to uncomment them. Fixed!